### ISSUE TITLE
atoms still work when not placed within a Provider Context

### ISSUE THREAD

**Issue by zcaudate** [pmndrs/jotai#469]
I tried a small experiment by not using Provider Context and seeing if the atoms are updating. It turns out that they do. 

Is there some sort of a Global Provider context for atoms and if so what does the use of the scoped Provider context do?

**Comment by zcaudate** [comment]
Oh...  so `Provider` allow atoms to be localised and effectively model local state. 

That's a really cool design. Thanks!

In terms of design, when would one choose to go the `Provider` with atoms option over just using `useState`?

**Comment by zcaudate** [comment]
yep. that makes sense.

I built a little global state library before discovering jotai and from my limited experience, localised prop drilling is the way to go for pure ui components like buttons and menus (I get mount/dismount errors if I use too much global state) but for modules needing data coordination (like forms), it's actually better to go with some sort of global state.

So it's either local state or global state for me and right nowI can't think of scenario where I'd use it for now as the default global store serves all my needs. But it's definitely a nice option to have.

**Comment by dai-shi** [comment]
Yeah, for your use cases, you wouldn't need Provider. It will be required for some special use cases like SSR or library usage.
